src/main/java/org/fcrepo/auth/webac/WebACAuthorization.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.auth.webac;
+
+import java.net.URI;
+import java.util.Set;
+
+import org.fcrepo.kernel.api.models.FedoraResource;
+
+import com.hp.hpl.jena.vocabulary.RDF;
+
+/**
+ * Interface for WebAC ACL objects.
+ * @author whikloj
+ * @since 2015-08-25
+ */
+public interface WebACAuthorization {
+
+    /*
+     * Set/Modify the acl:agents of this ACL.
+     * @param users a Set of agents
+     */
+    void setUsers(Set<String> users);
+
+    /**
+     * Get the set of acl:agents, empty set if none.
+     *
+     * @return set of agents
+     */
+    Set<String> getUsers();
+
+    /*
+     * Set/Modify the acl:agentClass of this ACL.
+     * @param users a Set of agentClasses
+     */
+    void setGroups(Set<URI> groups);
```

Comment:
I do not think our notion of "groups" is the same as `acl:agentClass`. Although the `Set<URI>` parameter to this method seems inline with the spec, the method name is confusing.